config/samples/istio-sample-kubernetes.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: istio-sample
 spec:
   version: v1.20.0
+  namespace: istio-system
```

Comment:
Should this field default to `istio-system`? In production systems, customers will want to use a custom namespace, but for first-time dev users, it might make sense to default to `istio-system`.

Reply:
yes, this is just a sample. All istio docs refer to istio-system in their samples. We are good.

Reply:
No, I mean in the CRD itself. Currently, the `namespace` field is required. But we could make it optional and default it to `istio-system`.

Reply:
Please no. Let users explicitly set it.

Reply:
Well, they don't have to specify it when they use `istioctl install`.

Reply:
Just because upstream does something it doesn't mean we have to blindly do it as well. We have to chance to "start over" with best practices from the beginning. IMO it's better to be explicit than implicit for sensitive fields like this. Specially if we plan to support any sort of multi-tenancy since the beginning. Just my $0.02.

Reply:
I'm not doing it blindly. I explain why it defaulting to `istio-system` would be a good idea.